api-docs/cloud-load-balancers-v1/api-reference/methods/put-update-ssl-termination-configuration-v1.0-account-loadbalancers-loadbalancerid-ssltermination.rst

```diff
@@ -226,13 +226,36 @@ The following table shows the body parameters for the request:
 |                          |                         |``ENABLED``. The default |
 |                          |                         |value is ``ENABLED``.    |
 +--------------------------+-------------------------+-------------------------+
+|cipherProfile             |String                   |Specifies a cipher       |
+|                          |                         |profile to be used. The  |
+|                          |                         |default value is         |
+|                          |                         |``default``.             |
++--------------------------+-------------------------+-------------------------+
+
+.. note::
+
+   CipherProfile's are defined by the service and are immutable.
+   The service currently offers two profiles. The 'default' profile, which
```

Comment:
The ``default`` profile is set on any load balancer enabling SSL termination by default.

Reply:
Removed this entire section.